demos/ASP.Net Core/ITOps.ViewModelComposition/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace ITOps.ViewModelComposition
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static void AddViewModelComposition(this IServiceCollection services, string assemblySearchPattern = "*ViewModelComposition*.dll")
```

Comment:
the parameter is there to allow people start playing with samples and not being forced to use our convention to name assemblies. I'll make it an overload.